Source/Details/UIView+ASConvenience.h

```diff
@@ -82,12 +82,15 @@ NS_ASSUME_NONNULL_BEGIN
  as they are already on NSObject
 
  @property (nonatomic, assign)           BOOL isAccessibilityElement;
- @property (nonatomic, copy)             NSString *accessibilityLabel;
- @property (nonatomic, copy)             NSString *accessibilityHint;
- @property (nonatomic, copy)             NSString *accessibilityValue;
+ @property (nonatomic, copy, nullable)   NSString *accessibilityLabel;
+ @property (nullable, nonatomic, copy)   NSAttributedString *accessibilityAttributedLabel API_AVAILABLE(ios(11.0),tvos(11.0));
+ @property (nonatomic, copy, nullable)   NSString *accessibilityHint;
+ @property (nullable, nonatomic, copy)   NSAttributedString *accessibilityAttributedHint API_AVAILABLE(ios(11.0),tvos(11.0));
+ @property (nonatomic, copy, nullable)   NSString *accessibilityValue;
+ @property (nullable, nonatomic, copy)   NSAttributedString *accessibilityAttributedValue API_AVAILABLE(ios(11.0),tvos(11.0));
```

Comment:
Please follow the `(nonatomic, copy, nullable)` convention for changes in this file.